app/wama.web/Controllers/CheckInController.cs

```diff
@@ -76,9 +87,9 @@ public IActionResult WaiverNotSigned()
         {
             var userAccount = await _UserAccountService.GetUserAccountAsync(memberId);
 
-            string userName = userAccount.FirstName + " " + userAccount.LastName;
+            string userName = userAccount.FirstName + " " + userAccount.LastName;            
```

Comment:
Don't just assume that the userAccount will always exist. 
Something weird can happen en-route to the DB, hence resulting in null being returned.

Perform proper checks for null on this variable.